src/webui/www/private/scripts/dynamicTable.js

```diff
@@ -1712,7 +1829,7 @@ const TorrentFilesTable = new Class({
             const progressBar = $('pbf_' + id);
             if (progressBar === null) {
                 td.adopt(new ProgressBar(value.toFloat(), {
-                    'id': 'pbf_' + id,
+                    'id': `pbf_${id}`,
```

Comment:
This is ES2015 template literals right? IE doesn't support it.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals#Browser_compatibility

Reply:
Good catch, will switch them back